introduction.md

```diff
@@ -1,2 +1,8 @@
 I am Sean Sweeney.
-My favorite Codeschool path is Javascript.
\ No newline at end of file
+My favorite Codeschool path is Javascript.
+
+Deadly Skills:  
+* Javascript  
+* GIS  
+* Python  
+* Bash 
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?